pkg/cmd/pipelinerun/testdata/TestListPipelineRuns-all_in_namespace.golden

```diff
@@ -1,6 +1,6 @@
-NAME    STARTED          DURATION   STATUS               
```

Comment:
The file name doesn't match the test name here, so I am assuming this is not being tested properly. Should be namespaces: `TestListPipelineRuns-all_in_namespaces.golden`